.github/workflows/covector-version-or-publish.yml

```diff
@@ -51,57 +51,31 @@ jobs:
           labels: "version updates"
           body: ${{ steps.covector.outputs.change }}
 
-  nodejs-binding-prebuild:
-    runs-on: ${{ matrix.os }}
+  binding-prebuild:
+    runs-on: ubuntu-20.04
     needs: version-or-publish
     if: needs.version-or-publish.outputs.successfulPublish == 'true'
     strategy:
       fail-fast: false
-      matrix:
-        # The GitHub hosted Windows 2022 image comes with Visual Studio 2022, but node-gyp
-        # (which is used by neon-sys) sadly fails to recognize it. As a mitigation, we still run the
-        # tests on Windows 2019, until we can figure out a way to fix the problem.
-        os: [ubuntu-latest, macos-latest, windows-2019]
-        node-version: ['14.x', '16.x', '18.x']
-        exclude:
-          - os: ubuntu-latest
-            node-version: '14.x'
 
     steps:
       - uses: actions/checkout@v3
 
-      - name: Set up Node.js ${{ matrix.node-version }}
+      - name: Set up Node.js 16.x
         uses: actions/setup-node@v2
         with:
-          node-version: ${{ matrix.node-version }}
+          node-version: 16.x
 
       - name: Install Rust toolchain
         uses: actions-rs/toolchain@v1
         with:
           toolchain: stable
           override: true
       
-      - name: Install LLVM and Clang (Windows) # required for bindgen to work, see https://github.com/rust-lang/rust-bindgen/issues/1797
```

Comment:
That's really not required anymore?

Reply:
It is required. Only the different node versions can be omitted.